trade-finance-manager-ui/server/controllers/utilisation-reports/index.js

```diff
@@ -0,0 +1,23 @@
+const { getFormattedReportDueDate, getFormattedReportPeriod } = require('../../services/utilisation-report-service');
+
+const getUtilisationReports = async (req, res) => {
+  try {
+    const { userToken } = req.session;
+
+    const reportPeriod = getFormattedReportPeriod();
+    const reportDueDate = await getFormattedReportDueDate(userToken);
+
+    return res.render('utilisation-reports/utilisation-reports.njk', {
+      activePrimaryNavigation: 'utilisation reports',
+      reportPeriod,
+      reportDueDate,
+    });
+  } catch (error) {
+    console.error('Error rendering utilisation reports page', error);
+    return res.redirect('/not-found');
```

Comment:
Can we re-introduce the 'something went wrong' page like we have in portal? Feels like `not-found` is dishonest from a user's point of view